server/routes/allCompletedReports/allCompletedReports.test.ts

```diff
@@ -95,7 +95,7 @@ describe('GET /all-completed-reports', () => {
         expect(res.text).not.toContain('All completed reports')
       })
   })
-  it('should render the correct page with the correct role', () => {
+  it.skip('should render the correct page with the correct role', () => {
```

Comment:
Also the test above this one checks for "all completed reports" - this should be changed to "Adjudications" as otherwise that one is a bit useless now - it'll never find "All completed reports"! :)

Reply:
Perhaps change the first test to
```
it('should render the not found page without the correct role', () => {
    userService.getUserRoles.mockResolvedValue([])
    return request(app)
      .get(adjudicationUrls.allCompletedReports.root)
      .expect('Content-Type', /html/)
      .expect(res => {
        expect(res.text).toContain('Page not found')
      })
  })
```
and second one to
```
  it('should render the correct page with the correct role', () => {
    userService.getUserRoles.mockResolvedValue(['ADJUDICATIONS_REVIEWER'])

    return request(app)
      .get(adjudicationUrls.allCompletedReports.root)
      .expect('Content-Type', /html/)
      .expect(res => {
        expect(res.text).toContain('Date of discovery')
      })
  })
```